Packages/Sites/Sfi.Kateheo/Resources/Private/Templates/NodeTypes/News.html

```diff
@@ -20,12 +20,16 @@ <h3 itemprop="headline">{neos:contentElement.editable(property: 'title', tag: 's
 							<meta itemprop="datePublished" content="{f:format.date(date:date, format:'Y-m-d')}" />
 						</time>
 					</span>
-					<f:if condition="{node.properties.author.properties.title}">
-						<!-- author -->
-						<span class="news-list-author" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person">
-							<span itemprop="name">{node.properties.author.properties.title}</span>
-						</span>
-					</f:if>
+					<f:for each="{authors}" as="item">
+						<f:if condition="{item.properties.title}">
+							<!-- author -->
+							<a href="/autors/{item.properties.uriPathSegment}">
```

Comment:
This is not a correct way to build links. See http://neos.readthedocs.org/en/stable/References/ViewHelpers/Neos.html#neos-link-node